hyperspy/learn/mva.py

```diff
@@ -1318,11 +1321,6 @@ def load(self, filename):
         if hasattr(self, 'ica_factors'):
             self.bss_factors = self.ica_factors
             del self.ica_factors
-        #
-        # Output_dimension is an array after loading, convert it to int
-        if hasattr(self, 'output_dimension') and self.output_dimension \
```

Comment:
Removed this check as it is included in the check [above](https://github.com/hyperspy/hyperspy/pull/2094/files#diff-cd68ac14189365f87b724293fa3c9f38R1285).